src/test/java/fi/otavanopisto/kuntaapi/test/ManagementMocker.java

```diff
@@ -4,33 +4,61 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
+
+import org.apache.commons.lang3.StringUtils;
 
 import fi.metatavu.management.client.model.Announcement;
+import fi.metatavu.management.client.model.Attachment;
 import fi.metatavu.management.client.model.Banner;
 import fi.metatavu.management.client.model.Fragment;
 import fi.metatavu.management.client.model.Menu;
+import fi.metatavu.management.client.model.Menuitem;
 import fi.metatavu.management.client.model.Page;
+import fi.metatavu.management.client.model.Pagemappings;
 import fi.metatavu.management.client.model.Post;
 import fi.metatavu.management.client.model.Tile;
 
 public class ManagementMocker extends AbstractMocker {
   
+  private static final String MEDIAS = "/wp-json/wp/v2/media";
   private static final String BANNERS = "/wp-json/wp/v2/banner";
   private static final String MENUS = "/wp-json/kunta-api/menus";
+  private static final String MENU_ITEMS = "/wp-json/kunta-api/menus/%s/items";
   private static final String PAGES = "/wp-json/wp/v2/pages";
   private static final String POSTS = "/wp-json/wp/v2/posts";
-  private static final String TILES = "/wp-json/wp/v2/tile";
+  private static final String TILES = "/wp-json/wp/v2/tile";;
   private static final String FRAGMENTS = "/wp-json/wp/v2/fragment";
   private static final String ANNOUNCEMENTS = "/wp-json/wp/v2/announcement";
+  private static final String PAGEMAPPINGS = "/wp-json/kunta-api/pagemappings";
   private static final String PATH_TEMPLATE = "%s/%s";
-
+  
+  private Pagemappings pagemappings = null;
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Rename field "pagemappings" to prevent any misunderstanding/clash with field "PAGEMAPPINGS" defined on line 33. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1845)